diff --git a/dist/connect.js b/dist/connect.js
index bf4beedd6617e13dbafab75c82f356095ffa4e5f..59f9fdc707832466aed1004e88a94a11c067df4e 100644
--- a/dist/connect.js
+++ b/dist/connect.js
@@ -35,6 +35,7 @@ const connectWithReconnect = async function({ adapter, pool, reconnect = false }
         // swallow error
         }
     });
+    result.release();
 };
 export const connect = async function connect(options = {
     hotReload: false
